### PR TITLE
Rubocop: Stay out of trouble

### PR DESCRIPTION
**Pull request by jaredmoody**
Another collection of rubocop autocorrected style fixes for #945

### CODE REVIEW

**Comment by jaredmoody** [comment]
Update: Dropped fixes for Style/Mutable constant since the tests failed. There must be constants being modified, I'll circle back to that after the autocorrect fixes are done.

**Comment by jasonrclark** [comment]
Looks like @brennx0r saw the same problem on #1253. Don't know if it's test or production code problem as yet.

**Comment by PragTob** [comment]
Not sure if I'm a fan of the `Style/Next` in general/in all its occurences - I try to avoid these loop special magics wherver I go and mostly they can be avoided with a select/reject beforehand. What do the others think?

File: samples/expert-othello-adjusted.rb [hunk 1]
```diff
@@ -262,14 +263,13 @@ def draw_board
             stroke rgb(0, 100, 0)
             rect left: left, top: top, width: PIECE_WIDTH, height: PIECE_HEIGHT
 
-            if cell != 0
-              strokewidth 0
-              fill(cell == 1 ? rgb(100,100,100) : rgb(155,155,155))
-              oval(left+3, top+4, PIECE_WIDTH-10, PIECE_HEIGHT-10)
+            next unless cell != 0
```
Comment (jasonrclark) [inline samples/expert-othello-adjusted.rb]: Yeah, for this sample in particular I feel like the `next` isn't nearly as obvious. I'd be onboard with disabling that particular rule.

**Comment by jasonrclark** [comment]
Hey @jaredmoody! Seems like @PragTob and I are in agreement that we'd rather skip on `Style/Next` but otherwise this PR looks good. Would you like to revert that yourself, or shall I go ahead with revising? Happy either way.

Thanks again for your continued tidying up!

**Comment by jaredmoody** [comment]
I'll revise, no problem.

**Comment by jaredmoody** [comment]
OK, this should be good to go as soon as Travis passes.

Also I had to go back and fix regressions introduced by other PRs. Do you guys think we could add rubocop to the CIs sooner rather than later to prevent that?

**Comment by jasonrclark** [comment]
Happy to get that added @jaredmoody. Feel free to make the change, or I'll get to it on my next Shoes break. Sorry for whatever I broke on you! 😢

**Pull request merged**